### DISCUSSION TITLE
Suggestions with white color | PyCharm

### DISCUSSION THREAD

**Discussion by WhiteWolfDD** [community/community#28915] [Copilot Conversations]
I have new issue. 
After updating Github Copilot I have white color for suggestion, not gray what before.

I marked down "Color for completions" and set the gray color. Restarted IDE and nothing.
![image](https://user-images.githubusercontent.com/71248043/183730092-cb64f991-350d-4e93-bc8c-c5f5efc2aef8.png)

The problem
![image](https://user-images.githubusercontent.com/71248043/183730378-e1285fb3-8690-46ae-92f5-21cf17f95379.png)

**Comment by WhiteWolfDD** [comment]
I noticed that the GitHub Copilot suggestions use the same color as the IDE's "Default text" color

**Reply by WhiteWolfDD** [reply]
![image](https://user-images.githubusercontent.com/71248043/183733827-52f54ac0-bf18-445a-a09b-237bb6e4f9f5.png)

*suggestion*
![image](https://user-images.githubusercontent.com/71248043/183733855-c3cf6468-110d-4350-8b6a-56245ba81a52.png)

**Comment by Plenituz** [comment]
Same issue with Goland #GO-222.3345.146 since the last update of the github copilot plugin yesterday evening (1.1.29.1869 I think), autocompletions are all white and "Color for completions" doesn't seem to apply even after restart

**Comment by CCXXXI** [comment]
I have seen more colors:

<img width="256" alt="image" src="https://user-images.githubusercontent.com/25550534/183841506-3b2b5048-50da-4735-8b8c-2e82c4f685d2.png">
<img width="256" alt="image" src="https://user-images.githubusercontent.com/25550534/183844030-cbc78ee1-e6ea-45c3-b570-9d0f6073454b.png">
<img width="256" alt="image" src="https://user-images.githubusercontent.com/25550534/183844256-7925aaa1-02a0-45dc-b650-77d0d9dcac98.png">

**Comment by WhiteWolfDD** [comment]
**Waiting for new update** 👍🏻

**Comment by mmelikyan** [comment]
And we're expected to pay for this product that doesn't even have proper QA? Come on, github.

Here's workaround by downgrading until Github fixes it.

Go to https://plugins.jetbrains.com/plugin/17718-github-copilot/versions/stable

Download version [1.1.28.1744](https://plugins.jetbrains.com/plugin/17718-github-copilot/versions/stable/190492)

Then install it from plugins. You might want to disable auto-updates.

<img width="593" alt="image" src="https://user-images.githubusercontent.com/108238904/183992286-4cbd583c-3d68-4e47-a638-4a1771d24a69.png">

**Reply by markwillis** [reply]
Thanks for the workaround, appreciated.

Given it's saved me 300+ hours in the week since I've been using it while coding blindfolded, I'm easily forgiving QA for the minor inconvenience. It's the QA on **_my_** team that should be worried. 

![blindfolded-typing](https://user-images.githubusercontent.com/43969188/184015975-cf6b8ee1-54ed-46d6-8bc2-afa990b827ed.gif)

**Reply by Tegasus9** [reply]
Thanks for your advice. Now I downgraded this plugin and it works great as before.

**Comment by WMOH-DEV** [comment]
Downgraded the plugin until the issue to be resolved

**Comment by edgarsi** [comment]
Apparently fails in all JetBrains IDEs, Android Studio #AI-212.5712.43.2112.8815526

**Comment by JohnCover** [comment]
Same issue in IntelliJ

**Comment by matteocontrini** [comment]
Fixed in 1.1.30.1887, released on 2022-08-17.
